src/test/java/calculator/Numbers.java

```diff
@@ -0,0 +1,26 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Numbers {
+
+    private List<PositiveNumber> numbers;
+
+    private Numbers(String[] numbers) {
+        this.numbers = Arrays.stream(numbers)
+                .map(number -> PositiveNumber.create(Integer.parseInt(number)))
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
                .map(number -> PositiveNumber.create(number))
```

Reply:
생성자를 추가하여 유연성을 높이도록 하였습니다.
정적팩토리메서드나 생성자를 호출해서 사용할때 호출시에 데이터타입을 맞춰서 호출을 하였었는데
생성자를 늘려서 유연해지도록 해보겠습니다.